Docker base image causes gunicorn error

```
2023-11-10 12:05:25 +0100] [2] [ERROR] Worker (pid:5169) was sent SIGINT!
[2023-11-10 12:05:25 +0100] [5171] [INFO] Booting worker with pid: 5171
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC 63186 current, 122600 max
[2023-11-10 12:05:25 +0100] [2] [ERROR] Worker (pid:5170) was sent SIGINT!
[2023-11-10 12:05:25 +0100] [5172] [INFO] Booting worker with pid: 5172
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC 63186 current, 122600 max
[2023-11-10 12:05:26 +0100] [2] [ERROR] Worker (pid:5171) was sent SIGINT!
[2023-11-10 12:05:26 +0100] [5173] [INFO] Booting worker with pid: 5173
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC 63186 current, 122600 max
[2023-11-10 12:05:26 +0100] [2] [ERROR] Worker (pid:5172) was sent SIGINT!
[2023-11-10 12:05:26 +0100] [5174] [INFO] Booting worker with pid: 5174
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC 63186 current, 122600 max
[2023-11-10 12:05:26 +0100] [2] [ERROR] Worker (pid:5173) was sent SIGINT!
[2023-11-10 12:05:26 +0100] [5175] [INFO] Booting worker with pid: 5175
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC 63186 current, 122600 max
[2023-11-10 12:05:27 +0100] [2] [ERROR] Worker (pid:5174) was sent SIGINT!
[2023-11-10 12:05:27 +0100] [5176] [INFO] Booting worker with pid: 5176
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC 63186 current, 122600 max
[2023-11-10 12:05:27 +0100] [2] [ERROR] Worker (pid:5175) was sent SIGINT!
[2023-11-10 12:05:27 +0100] [5177] [INFO] Booting worker with pid: 5177
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC 63186 current, 122600 max
[2023-11-10 12:05:27 +0100] [2] [ERROR] Worker (pid:5177) was sent SIGINT!
[2023-11-10 12:05:27 +0100] [5178] [INFO] Booting worker with pid: 5178
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC 63186 current, 122600 max
[2023-11-10 12:05:28 +0100] [2] [ERROR] Worker (pid:5176) was sent SIGINT!
[2023-11-10 12:05:28 +0100] [5179] [INFO] Booting worker with pid: 5179
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC 63186 current, 122600 max
[2023-11-10 12:05:28 +0100] [2] [ERROR] Worker (pid:5178) was sent SIGINT!
[2023-11-10 12:05:28 +0100] [5180] [INFO] Booting worker with pid: 5180
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC 63186 current, 122600 max
[2023-11-10 12:05:28 +0100] [2] [ERROR] Worker (pid:5179) was sent SIGINT!
[2023-11-10 12:05:28 +0100] [5181] [INFO] Booting worker with pid: 5181
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC 63186 current, 122600 max
[2023-11-10 12:05:28 +0100] [2] [ERROR] Worker (pid:5180) was sent SIGINT!
[2023-11-10 12:05:28 +0100] [5182] [INFO] Booting worker with pid: 5182
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC 63186 current, 122600 max
[2023-11-10 12:05:29 +0100] [2] [ERROR] Worker (pid:5181) was sent SIGINT!
[2023-11-10 12:05:29 +0100] [5183] [INFO] Booting worker with pid: 5183
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC 63186 current, 122600 max
[2023-11-10 12:05:29 +0100] [2] [ERROR] Worker (pid:5182) was sent SIGINT!
[2023-11-10 12:05:29 +0100] [5184] [INFO] Booting worker with pid: 5184
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC 63186 current, 122600 max
[2023-11-10 12:05:30 +0100] [2] [ERROR] Worker (pid:5183) was sent SIGINT!
[2023-11-10 12:05:30 +0100] [5185] [INFO] Booting worker with pid: 5185
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC 63186 current, 122600 max
[2023-11-10 12:05:30 +0100] [2] [ERROR] Worker (pid:5184) was sent SIGINT!
[2023-11-10 12:05:30 +0100] [5186] [INFO] Booting worker with pid: 5186
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC 63186 current, 122600 max
[2023-11-10 12:05:30 +0100] [2] [ERROR] Worker (pid:5185) was sent SIGINT!
[2023-11-10 12:05:30 +0100] [5187] [INFO] Booting worker with pid: 5187
OpenBLAS blas_thread_init: pthread_create failed for thread 1 of 8: Operation not permitted
OpenBLAS blas_thread_init: RLIMIT_NPROC 63186 current, 122600 max
[2023-11-10 12:05:30 +0100] [2] [ERROR] Worker (pid:5186) was sent SIGINT!
[2023-11-10 12:05:31 +0100] [5188] [INFO] Booting worker with pid: 5188
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.1.12]
+### Fixed
+-  Modified Docker files to use python:3.9-slim-bullseye to prevent gunicorn workers booting error
+
 ## [0.1.11]
 ### Fixed
 -  For compliance with latest MongoDB versions update loqusdb version to >=2.7.3 in requirements

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM python:3.9-slim
+FROM python:3.9-slim-bullseye
 
 ENV GUNICORN_WORKERS=1
 ENV GUNICORN_THREADS=1

